### PR TITLE
Update RabbitMQ docs to core V8

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+partial class Upgrade
+{
+    void CertificatePath(EndpointConfiguration endpointConfiguration)
+    {
+        #region 6to7certificatepath7
+
+        var transport = new RabbitMQTransport(Topology.Conventional, "host=localhost");
+
+        transport.ClientCertificate = new X509Certificate2("path", "password");
```
Comment (timbussmann) [inline Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs:12]: use object initializers syntax instead?

File: transports/rabbitmq/connection-settings_prefetch-control_rabbit_[7,).partial.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Controlling the prefetch count
+
+When consuming messages from the broker, throughput can be improved by having the consumer [prefetch](https://www.rabbitmq.com/consumer-prefetch.html) additional messages.
+By default the prefetch count is calculated by multiplying [maximum concurrency](/nservicebus/operations/tuning.md#tuning-concurrency) by the prefetch multiplier of 3. The default prefetch count calculation can be overridden. It can still take into account the configured concurrency settings that is passed as the input to the calculation function.
```
Comment (timbussmann) [inline transports/rabbitmq/connection-settings_prefetch-control_rabbit_[7,).partial.md:4]:
```suggestion
By default the prefetch count is calculated by multiplying [maximum concurrency](/nservicebus/operations/tuning.md#tuning-concurrency) by the prefetch multiplier of 3. The default prefetch count calculation can be overridden with a custom algorithm. The configured concurrency setting is passed as an input to the calculation function.
```

File: transports/upgrades/rabbitmq-6to7.md [hunk 3]
```diff
@@ -15,3 +15,63 @@ upgradeGuideCoreVersions:
 ## Timeout manager
 
 The [timeout manager is removed from core](/nservicebus/upgrades/7to8/#timeout-manager-removed) which makes timeout manager backwards compatibility mode obsolete. If backwards compatibility mode was enabled these APIs must be removed.
+
+## Configuring RabbitMQ Transport
+
+To use Azure Service Bus Transport for NServiceBus, create a new instance `AzureServiceBusTransport` and pass it to `EndpointConfiguration.UseTransport`.
```
Comment (mauroservienti) [inline transports/upgrades/rabbitmq-6to7.md:21]:
```suggestion
To use the RabbitMQ transport for NServiceBus, create a new instance of the `RabbitMQTransport` and pass it to `EndpointConfiguration.UseTransport`.
```

File: transports/upgrades/rabbitmq-6to7.md [hunk 4]
```diff
@@ -15,3 +15,63 @@ upgradeGuideCoreVersions:
...
+
+snippet: 6to7prefetchcount7
+
+## Disabling the durable exchanges and queues
+
+Disabling the durable exchanges and queues has been moved to the constructor of the topology classes, `ConventionalRoutingTopology` and `DirectRoutingTopology`. In order to set the value of that parameter use the variant of the `RabbitMQTransport` constructor that accepts an instance of the topology.
+
+## Configuration Options
+
+The RabbitMQ Transport configuration options that have not been changed have been moved to the `RabbitMQTransport` class. See the following table for further information:
```
Comment (mauroservienti) [inline transports/upgrades/rabbitmq-6to7.md:67]:
```suggestion
The RabbitMQ transport configuration options that have not been changed have been moved to the `RabbitMQTransport` class. See the following table for further information:
```

**Review by mauroservienti — CHANGES_REQUESTED** [review]

File: Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs (hunk 1 above)
Comment (mauroservienti) [inline Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs:12]:
```suggestion
        var transport = new RabbitMQTransport(Topology.Conventional, "host=localhost")
        {
            ClientCertificate = new X509Certificate2("path", "password")
        };
```

File: Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    void PrefetchCount(EndpointConfiguration endpointConfiguration)
+    {
+        #region 6to7prefetchcount7
+
+        var transport = new RabbitMQTransport(Topology.Conventional, "host=localhost");
+
+        transport.PrefetchCountCalculation = _ => 100;
+        //or
+        transport.PrefetchCountCalculation = concurrency => concurrency * 7;
```
Comment (mauroservienti) [inline Snippets/Rabbit/Rabbit_7/UpgradeGuides/6to7/Upgrade.cs:27]:
```suggestion
        var transportWithFixedPrefetchCount = new RabbitMQTransport(Topology.Conventional, "host=localhost")
        {
            PrefetchCountCalculation = _ => 100
        };

        //or

        var transportWithConcurrencyBasedPrefetchCount = new RabbitMQTransport(Topology.Conventional, "host=localhost")
        {
            PrefetchCountCalculation = concurrency => concurrency * 7
        };
```

**Review by mauroservienti — APPROVED** [review]
Does it make sense to change also these two?

**Pull request merged**